Firebase/Messaging/FIRMessaging.m

```diff
@@ -541,9 +541,20 @@ - (void)setAPNSToken:(NSData *)apnsToken type:(FIRMessagingAPNSTokenType)type {
 #pragma mark - FCM
 
 - (BOOL)isAutoInitEnabled {
+  // Defer to the class method since we're just reading from regular userDefaults and we need to
+  // read this from IID without instantiating the Messaging singleton.
+  return [[self class] isAutoInitEnabledWithUserDefaults:_messagingUserDefaults];
+}
+
+/// Checks if Messaging auto-init is enabled in the user defaults instance passed in. This is
+/// exposed as a class property for IID to fetch the property without instantiating an instance of
+/// Messaging. Since Messaging can only be used with the default FIRApp, we can have one point of
+/// entry without context of which FIRApp instance is being used.
+/// ** THIS METHOD IS DEPENDED ON INTERNALLY BY IID. PLEASE DO NOT CHANGE THE SIGNATURE. **
```

Comment:
From my read it looks like `isAutoInitEnabled` is depended upon by Messaging. `isAutoInitEnabledWithUserDefaults` is only depended upon by the tests.

Reply:
nvm, I missed the call.

Reply:
This comment doesn't strongly enough indicate that the call is made reflectively so it won't necessarily show up in find usages.